css/id.css

```diff
@@ -2,7 +2,7 @@
     background: url("../images/nrg_bg.jpg");
     background-repeat: no-repeat;
     background-size: cover;
-    height: calc(100vh - 303px);
+    height: 100%;
```

Comment:
This actually causes a weird white gap in the home page (https://puu.sh/vSuPL/31d86d4858.png). Consider making it greater than 100% or keeping the old one which calculates the vertical height minus the header/footer size.